### PR TITLE
Network error proper handling

### PR DESCRIPTION
**Pull request by YaroTs**

### CODE REVIEW

File: mobile/src/people/avatar.tsx [hunk 1]
```diff
@@ -104,7 +104,7 @@ class AvatarImpl extends Component<AvatarProps, AvatarState> {
             this.props.imageStyle
         ]) as ImageStyle;
 
-        if (!(this.props.photoUrl && this.props.jwtToken) || this.state.loadingErrorOccurred) {
+        if (!(this.props.photoUrl) || this.state.loadingErrorOccurred) {
```
Comment (antonm76) [inline mobile/src/people/avatar.tsx]:
```suggestion
        if (!this.props.photoUrl || this.state.loadingErrorOccurred) {
```

File: mobile/src/auth/secured-api-client.ts [hunk 2]
```diff
@@ -43,7 +43,7 @@ export class SecuredApiClient {
 
     //----------------------------------------------------------------------------
     private isAuthenticated(state: AuthenticationState): state is AuthenticatedState {
-        if (state.isAuthenticated) {
+        if (state.isAuthenticated && state.jwtToken) {
```
Comment (antonm76) [inline mobile/src/auth/secured-api-client.ts]: What's happening without that `&& state.jwtToken` thing?

File: mobile/src/auth/oauth-process.ts [hunk 3]
```diff
@@ -200,6 +195,18 @@ export class OAuthProcess {
         this.refreshTokenSource.next(null);
         this.authenticationStateSource.next(errorInstance);
     }
+
```
Comment (antonm76) [inline mobile/src/auth/oauth-process.ts]:
```suggestion
//----------------------------------------------------------------------------
```

File: mobile/src/auth/oauth-process.ts [hunk 4]
```diff
@@ -200,6 +195,18 @@ export class OAuthProcess {
...
+    private isCancellationError(error: any): boolean {
+        const errorCode = error.code;
+
+        return errorCode && errorCode === cancellationErrorCode;
+    }
+
```
Comment (antonm76) [inline mobile/src/auth/oauth-process.ts]:
```suggestion
//----------------------------------------------------------------------------
```

File: mobile/src/people/avatar.tsx (hunk 1 above)
Comment (YaroTs) [reply mobile/src/people/avatar.tsx]: fbf01c48453e7760c25d3ad33301a63f48f65439

File: mobile/src/auth/oauth-process.ts (hunk 4 above)
Comment (YaroTs) [reply mobile/src/auth/oauth-process.ts]: fbf01c48453e7760c25d3ad33301a63f48f65439

**Pull request closed without merging**